### ISSUE TITLE
Toggle replacement text while logged out

### ISSUE THREAD

**Issue by rebeccacremona** [harvard-lil/h2o#653]
Replacement annotations, while closed, look like:
```
<span data-v-1a689b4a="" data-exclude-from-offset-calcs="true" class="replacement-text" data-content="Replacing text" contenteditable="false">Replacing text</span>
```
These should be considered buttons, buttons that can be pressed to toggle from the replaced text to the original text. The easiest solution is to use a "button" element", but that breaks the export code. See prod for a complete example of accessible markup that does not break the export code, which includes:
- markup to make the `span` keyboard focusable
- markup to make the `span` semantically a button
- markup to give the button an appropriate label
- javascript for forcing this retrofitted span to behave like a true button when enter and spacebar are pressed

I will add more concrete hints to this issue via comments below.

**Comment by rebeccacremona** [comment]
closed in favor of https://github.com/harvard-lil/h2o/issues/655
